dotCMS/src/main/java/com/dotcms/auth/providers/jwt/beans/UserToken.java

```diff
@@ -76,8 +77,8 @@ public UserToken(String id, String subject, Date modificationDate, long ttlMilli
      * @param subject - The subject of the token
      * @param ttlMillis - The expiration date of the token.
      */
-    public UserToken(String id, String subject, String issuer, Date modificationDate, long ttlMillis) {
-        this(id, subject, issuer, modificationDate,ttlMillis, ImmutableMap.of());
+    public UserToken(String id, String subject, String issuer, Date modificationDate, long ttlMillis, final String skinId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'ttlMillis' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)